external/js/cothority/src/skipchain/skipchain-rpc.ts

```diff
@@ -149,7 +149,7 @@ export default class SkipchainRPC {
      * @param verify    Verify the integrity of the chain when true
      * @returns a promise that resolves with the list of blocks
      */
-    async getUpdateChain(latestID: Buffer, verify = true): Promise<SkipBlock[]> {
+    async getUpdateChain(latestID: Buffer, verify = true, acceptForeigner = false): Promise<SkipBlock[]> {
```

Comment:
Here and in `getLatestBlock`:
`s/acceptForeigner/acceptProxy/`

we accept foreigners always, regardless of this setting ;)